client/src/components/GameImage/mobileStats.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Note } from '../typography';
+
+const Wrapper = styled.div`
+  padding: 1vw 0 0 3vw;
+  @media ${p => p.theme.media.landscape} {
+    padding: 1vh 0 0 3vh;
+  }
+
+  & > * {
+    margin-bottom: 1vh;
+  }
+`;
+
+const MobileStats = ({ yearPublished, rating, stats, ...props }) => {
+  const {
+    maxPlayers,
+    minPlayers,
+    suggestedPlayers,
+    maxPlaytime,
+    minPlaytime
+  } = stats;
+
+  return (
+    <Wrapper>
+      <div>
+        <div>
+          {minPlayers === maxPlayers
+            ? `${maxPlayers} players `
+            : `${minPlayers}-${maxPlayers} players`}
+        </div>
+        <div>
+          {suggestedPlayers &&
+            minPlayers !== maxPlayers &&
+            `best with ${suggestedPlayers}`}
+        </div>
+      </div>
+
+      <div>
+        plays in{' '}
+        {minPlaytime === maxPlaytime
+          ? maxPlaytime
+          : `${minPlaytime} - ${maxPlaytime}`}{' '}
+        minutes
+      </div>
```

Comment:
`{' '}` while these work they read a bit weird
```suggestion
      <div>
        {`plays in ${minPlaytime === maxPlaytime ? maxPlaytime : `${minPlaytime} - ${maxPlaytime}`} minutes`}
      </div>
```